core/domain/LDAStringClassifierTest.py

```diff
@@ -256,37 +257,52 @@ def test_validate(self):
         }
         self.classifier.validate(classifier_data)
 
-        # Verify Type error is raised when string is provided instead of int.
+        # Verify validation error is raised when key is not present.
+        classifier_data.pop('_alpha', None)
+        with self.assertRaisesRegexp(utils.ValidationError, "key"):
+            self.classifier.validate(classifier_data)
+
+        # Verify Validation error is raised when string is provided instead of
```

Comment:
This should be either "Verify that a ValidationError is raised" or "Verify that a validation error is raised", but don't mix-and-match. Ditto below.